src/services/ajv.ts

```diff
@@ -30,6 +32,7 @@ export const addKeywords = (ajv: any) => {
     ajv.addKeyword('advanced'); // Should be collapsed by default (over ridden if section contains required fields)
     ajv.addKeyword('order'); // Used to order the fields in the UI
     ajv.addKeyword('x-oauth2-provider'); // Used to display OAuth
+    ajv.addKeyword('x-collection-name'); // Used to default name in resource configs
```

Comment:
We keep all custom annotations in here so we can keep them all documented together.